virtme/architectures.py

```diff
@@ -256,9 +256,7 @@ class Arch_aarch64(Arch):
     def __init__(self):
         Arch.__init__(self, "aarch64")
 
-        self.qemuname = "aarch64"
         self.linuxname = "arm64"
-        self.gccname = "aarch64"
```

Comment:
I'm wondering if there's a better way to avoid removing gccname and qemuname, my concern is that we may break something if we drop these definitions...